data/chapters/austin.json

```diff
@@ -3,20 +3,20 @@
   "title": "Austin GiveCamp",
   "city": "Austin",
   "state": "Texas",
-  "latitude": 30.4010897,
-  "longitude": -97.7517582,
+  "latitude": 30.265886,
+  "longitude": -97.749298,
   "img": "austin.jpg",
   "img_attribution": "iStock",
   "events": [
     {
       "start_date": "2016-11-04",
       "end_date": "2016-11-06",
-      "street": "10415 Morado Circle Bld 3 Ste 100",
+      "street": "500 W 2nd St",
       "city": "Austin",
       "state": "Texas",
-      "postal_code": 78759,
-      "volunteer_signup_url": null,
-      "non_profit_signup_url": null
+      "postal_code": 78701,
+      "volunteer_signup_url": "https://www.austingivecamp.org/non-profit-form",
+      "non_profit_signup_url": "https://www.austingivecamp.org/volunteer-form"
     }
```

Comment:
Is this to update the 2016 event, or to add the 2019 event? If adding a new event, add another object to the JSON array:

```
"events": [
    {
      "start_date": "2016-11-04",
      "end_date": "2016-11-06",
      "street": "10415 Morado Circle Bld 3 Ste 100",
      "city": "Austin",
      "state": "Texas",
      "postal_code": 78759,
      "volunteer_signup_url": null,
      "non_profit_signup_url": null
    },
    {
      "start_date": "2019-01-01",
      "end_date": "2019-01-03",
      "street": "500 W 2nd St",
      "city": "Austin",
      "state": "Texas",
      "postal_code": 78701,
      "volunteer_signup_url": "https://www.austingivecamp.org/non-profit-form",
      "non_profit_signup_url": "https://www.austingivecamp.org/volunteer-form"
    }
  ]
```

Be sure to change the start and end dates for your 2019 event.